sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/complexTypeCreator.scala

```diff
@@ -52,7 +52,8 @@ trait NoThrow
       > SELECT _FUNC_(1, 2, 3);
        [1,2,3]
   """,
-  since = "1.1.0")
+  since = "1.1.0",
+  group = "array_funcs")
```

Comment:
Could you clarify that the difference between `array_funcs` and `collection_funcs` in this context?